extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -59,12 +60,13 @@ export const StateProperty = {
   AD_STATE: 'adstate',
   BOOKEND_STATE: 'bookendstate',
   DESKTOP_STATE: 'desktopstate',
-  HAS_AUDIO_STATE: 'hasaudiostate',
+  STORY_HAS_AUDIO_STATE: 'storyaudiostate',
   LANDSCAPE_STATE: 'landscapestate',
   MUTED_STATE: 'mutedstate',
+  PAGE_HAS_AUDIO_STATE: 'pageaudiostate',
   SHARE_MENU_STATE: 'sharemenustate',
   SUPPORTED_BROWSER_STATE: 'supportedbrowserstate',
-  CURRENT_PAGE_ID: 'currentpageid',
+  CURRENT_PAGE_ID: 'currentpageid',  
```

Comment:
nit: trailing whitespace here
just a tip: you can add a macro to your editor so that when you save a file it gets rid of all trailing whitespace. I had the same issue:
https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut

Reply:
That's really helpful! Thanks!